src/hooks/useBaseURL.js

```diff
@@ -0,0 +1,11 @@
+export const useBaseURL = () => {
```

Comment:
This custom hook doesn't have a state or calls any other hook, so you can have this as a normal JS function.